third_party/txt/src/txt/platform_linux.cc

```diff
@@ -13,84 +13,14 @@
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
-#ifdef FLUTTER_USE_FONTCONFIG
   return {"TizenDefaultFont"};
```

Comment:
This is only valid when using font config at tizen device. So, if you are going to use this, we will have to use the font config as the default and delete other define(FLUTTER_USE_FONTCONFIG).

Reply:
As we have checked now, in the case of TV, SamsungOneUI is necessary. :anger:

Reply:
Oh.. I will add "SamsungOneUI". But why? Could you give me the detailed reason?

Reply:
Thanks for the fix! This is great. Now we can finally see SamsungOneUI in all devices.

> Oh.. I will add "SamsungOneUI". But why? Could you give me the detailed reason?

I think its because a certain setting file of fontconfig in `/etc/fonts/conf.d` sets the name of default font as "SamsungOneUI" on TV. If I remember correctly, I think the file name contained tizen something.

Reply:
The file is `/etc/fonts/conf.d/99-slp.conf`.

I've checked with mobile and wearable emulators, they do work with "TizenDefaultFont".

Reply:
To add a little...
You can find conf file for fontconfig at below:
https://review.tizen.org/gerrit/#/admin/projects/platform/core/graphics/default-fonts-fc-sdk
All platforms except TV use what is in there.
In case of TV, use
```
99-slp.conf
```
It seems to be a pretty old file & It is maintained only through updating without replacing new one.